### ISSUE TITLE
Is this a false positive for ARIA [role = "doc-endnotes"]?

### ISSUE THREAD

**Issue by andreseduardop** [GoogleChrome/lighthouse#12432]
Lighthouse encounters the following Accessibility error:
_"Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children."_
On the page: [https://lectupedia.com/en/number-of-books-read-in-each-country/](https://lectupedia.com/en/number-of-books-read-in-each-country/)

However, the page contains a `<section>` with `role="doc-endnotes"` and several children of type `<li>` with `role="doc-endnote"`

**Comment by adamraine** [comment]
I am noticing this with the Axe extension as well so we should move this issue to https://github.com/dequelabs/axe-core. I think https://github.com/dequelabs/axe-core/issues/2759 captures a similar bug.

Interestingly, I came across [this](http://kb.daisy.org/publishing/docs/html/dpub-aria/doc-endnotes.html) which says the violation should be ignored.

**Comment by andreseduardop** [comment]
Thanks, @adamraine

**Comment by davidmerfield** [comment]
I'm unsure why this was closed – the issue does not seem to be solved – I understand it's a problem with a dependency. At the moment lighthouse will complain if you pass it the [example copied directly from the W3G spec](https://www.w3.org/TR/dpub-aria-1.0/#doc-endnote):

```html
<section role="doc-endnotes">
   <h2>Notes</h2>
   <ol>
      <li id="6baa07af" role="doc-endnote">Additional results of this study can be found at … </li>
      <li id="7b2c0555" role="doc-endnote">…</li>
      …
   </ol>
</section>
```

![image](https://user-images.githubusercontent.com/747928/134078513-19a10dd4-1472-42e8-a559-d11e75ecf49a.png)

**Comment by adamraine** [comment]
Looks like the issue was closed in axe-core 4.4. We will need to bump our dep.

**Comment by Seirdy** [comment]
> Looks like the issue will be closed in axe-core 4.4. We will need to bump our dep when that is released.

@adamraine Axe 4.4.1 has been released, and deprecates doc-endnote.
